packages/shared/src/components/profile/UserMetadata.tsx

```diff
@@ -69,6 +69,7 @@ export function UserMetadata({
       {!!company && (
         <div className="mt-4 flex items-center gap-1">
           <ProfilePicture
+            className="border border-border-subtlest-secondary"
```

Comment:
Ah this one can actually use the other one i guess 🙈
Not sure why i made 2 different ones (if not more)

Reply:
Don't know 🙈
Will take a look and open another PR if they're mergeable